### DISCUSSION TITLE
add info in myprofile

### DISCUSSION THREAD

**Discussion by zicohip** [community/community#61518] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

how can i add this in my profile 

![image](https://github.com/community/community/assets/76248189/f70fc434-a789-4e1d-932e-3e7b588ca0bb)

**Comment by ahmedna126** [comment, marked as answer]
Create a repository so that the repository name is the same as your username

In the README.md file add your information
